crontab

```diff
@@ -7,11 +7,11 @@
 15 20 * * * bundle exec rake not_ready_reminders:remind
 */5 * * * * bundle exec rake worker_heartbeat:perform
 */10 * * * 1-6 bundle exec rake efile:poll_and_get_acknowledgments
```

Comment:
Do we have another ticket to comment out/remove this after we stop filing on 10/31 (state_file_end_of_in_progress_intakes)?